docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    plt.rcParams["figure.figsize"] = [20, 4]  # width, height
</pre><p>You can replace these with <code>plt.style.use('stumpy.mplstyle')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good catch - thank you!